### PR TITLE
New background for highlighted comments

### PR DESCRIPTION
**Pull request by yanncloud**
- delete old background
- highlight commentContent with editor-primary-100

### CODE REVIEW

**Comment by yanncloud** [comment]
@elbotho Ich habe vercheckt, wie ich dich als Reviewer hinzufügen kann. 😅 Ich dachte, dass kann ich noch nach dem Pull Request machen.

**Comment by Entkenntnis** [comment]
more people joining the frontend 🎉 🎉  :)

File: src/components/comments/comment.tsx [hunk 1]
```diff
@@ -80,7 +80,7 @@ export function Comment({
 
     if (commentRef.current) {
       commentRef.current.style.backgroundColor = isHighlight
-        ? '#fdf5e4'
+        ? ''
         : 'transparent'
     }
```
Comment (elbotho) [inline src/components/comments/comment.tsx]: @yanncloud I think this whole block is needed any more now, could you remove it?

**Review by elbotho — CHANGES_REQUESTED** [review]

**Comment by elbotho** [comment]
🎉 nicely done so far, I requested one more change and then we can merge it.

File: src/components/comments/comment.tsx (hunk 1 above)
Comment (yanncloud) [reply src/components/comments/comment.tsx]: of course :)

**Review by Entkenntnis — APPROVED** [review]
test for vercel integration

**Comment by elbotho** [comment]
@yanncloud Thank you! You can now inform Gregor and send him a link to https://de.serlo-staging.dev/discussions where he can review the changes (in some minutes).

**Pull request merged**